src/GraphQL.Server.AspNetCore.GraphiQL/IMiddleware.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+
+namespace GraphQL.Server.AspNetCore.GraphiQL {
```

Comment:
I think these are unnecessary abstractions on top of an existing abstractions. What value do they provide?

Reply:
It's not required, deleting it